server/server.js

```diff
@@ -4,18 +4,17 @@ const path = require('path');
 
 
 // use express.json instead of body-parser
-app.use(express.json())
+app.use(express.json());
 const PORT = 3000;
 
-
-app.get('/', (req, res) => {
-  res.sendFile(path.resolve(__dirname, './../index.html'));
+app.listen(PORT, () => {
+  console.log(`Listening on PORT: ${PORT}`);
 })
 
 
+app.get('/', (req, res) => {
```

Comment:
Did you accidentally switch the position of these code lines? app.listen should be at the bottom (maybe discard your changes on this file?)